### DISCUSSION TITLE
DHT12 incorrect temperature readings

### DISCUSSION THREAD

**Discussion by klepan63** [arendst/Tasmota#11381] [Q&A] [answered]
Hi!
on version 9.2.0 shows correctly

![dht12_0](https://user-images.githubusercontent.com/60322730/111692227-a7d97000-8848-11eb-9e92-54a95144b8e1.jpg)

after updating to 9.3.0 / 9.3.1 the temperature shows incorrectly

![dht12](https://user-images.githubusercontent.com/60322730/111691817-22ee5680-8848-11eb-8d75-b1ca7575a37a.jpg)

**Comment by ascillato** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Reply by ascillato** [reply]
Hi, there is no problem in Tasmota software with this sensor. Your problem is another issue. Not a question for discussions.

If you want help to troubleshoot your issue, please address this to the chat.

**Reply by klepan63** [reply]
Hi,
on the old firmware the sensor works, but on the new one it does not work and do you think that the problem is not in the firmware? 
PS: in this chat I was advised to change the sensor to a more modern one

**Reply by ascillato** [reply]
Please, open a new issue with the troubleshooting template completed in order to not need to ask all those standard questions here. Discussions is meant for talk about a topic (new feature request, modifications, Tasmota usage, questions, projects, etc) issues is better to be handled in github issues with the troubleshooting template completed or to be addressed in the chat. Thanks.

**Comment by 420pootang69** [comment]
Probably unrelated, I actually had this issue on the latest release with a DSB18B20 and it turned out to be a 3.3v supply that was dipping. Again, probably not related, but it wouldn't hurt to monitor the 3.3v rail for a few minutes to rule it out.
